tfhe/benches/integer/glwe_packing_compression.rs

```diff
@@ -169,6 +169,9 @@ mod cuda {
 
         let log_message_modulus = param.message_modulus.0.ilog2() as usize;
 
+        let cks = ClientKey::new(param);
```

Comment:
Since we are modifying tests and everything else to use multi-bit parameters within the GPU context, probably we should do the same here. Line 167 could be changed.

Reply:
Done